src/types/admin.ts

```diff
@@ -10,7 +10,7 @@ export type EditorsType = {
 
 export type ContentFeedbackArgs = {
   wikiId: string
-  userId: string
+  userId: string | undefined
```

Comment:
```suggestion
  userId?: string
```

You can use it like this.